app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= render @recipe %>
+
+<div class="mt-4 d-flex align-items-center gap-2">
+  <%= link_to "Edit this recipe", edit_recipe_path(@recipe), class: 'btn btn-primary mr-2' %>|
+  <%= link_to "Back to recipes", recipes_path, class: 'btn btn-secondary mr-2' %>|
+  <%= button_to "Delete this recipe", @recipe, method: :delete, data: { confirm: 'Are you sure?' }, class: 'btn btn-danger' %>
+</div>
```

Comment:
- [ ] Good job implementing the show page to display information about a particular recipe :+1: However, it would be nice to follow the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true) as requested in the project requirements.

- [ ] Please make sure that on the recipe show page, there is a button for adding ingredient to the recipe

- [ ] Kindly make sure that on the recipe show page, there is a button for generating a shopping list for the recipe and it matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/general_shopping_list.png?raw=true) and url `/general_shopping_list`